test/utils/dirContentsToObject.js

```diff
@@ -1,18 +1,20 @@
 const { promisify } = require('es6-promisify');
 const readFilesAsync = promisify(require('node-dir').readFiles);
-const { relative } = require('path');
+const { relative, extname } = require('path');
 
-module.exports = async dirname => {
+module.exports = async (dirname, includeExtensions) => {
   const files = {};
 
   const handleFile = (err, content, filePath, next) => {
     if (err) throw err;
 
     const relativeFilePath = relative(dirname, filePath);
     
-    files[relativeFilePath] = /\.js$/.test(relativeFilePath)
-      ? 'CONTENTS IGNORED IN SNAPSHOT TEST'
-      : content;
+    if (!includeExtensions || includeExtensions.includes(extname(relativeFilePath))) {
```

Comment:
This was added to include other files into the snapshot. Specifically the `.css.js.d.ts`files as they are generated to the `src` directory.